packages/web/src/tracer.ts

```diff
@@ -0,0 +1,45 @@
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web'
+import { DocumentLoadInstrumentation } from '@opentelemetry/instrumentation-document-load'
+import { FetchInstrumentation } from  '@opentelemetry/instrumentation-fetch'
+import { XMLHttpRequestInstrumentation } from  '@opentelemetry/instrumentation-xml-http-request'
+import { UserInteractionInstrumentation } from '@opentelemetry/instrumentation-user-interaction'
+import { ZoneContextManager } from '@opentelemetry/context-zone'
+import { registerInstrumentations } from '@opentelemetry/instrumentation'
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'
+
+// import { AudiusInstrumention } from '@audius/instrumentation-audius'
```

Comment:
Ideally yes, it'll be a separate published package that anyone can consume. To make things easier though, it might be better to just keep it in the same codebase for now